src/main/java/frc/robot/commands/AmpPrepScore.java

```diff
@@ -38,4 +40,17 @@ public AmpPrepScore(Elevator elevator, Conveyor conveyer, Intake intake) {
               return intake.atDesiredIntakePosition() && elevator.atDesiredPosition();
             }));
   }
+
+  /**
+   * runs AmpPrepScore but adds the layer that if the robot is interrupted, then everything is
+   * brought to the home state
+   */
+  public static Command interruptibleAmpPrepScore(
+      Intake intake, Elevator elevator, Conveyor conveyor, Shooter shooter) {
+    return new AmpPrepScore(intake, elevator, conveyor, shooter)
+        .finallyDo(
+            (boolean interrupted) -> {
+              new StopMostThings(intake, elevator, conveyor, shooter);
+            });
```

Comment:
This runs whether the command was interrupted or not, so it'll just immediately go home once it's done prepping score